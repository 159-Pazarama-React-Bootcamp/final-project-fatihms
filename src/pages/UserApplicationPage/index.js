import React from "react";

import Header from "../../components/Header";
import ApplicationInfo from "../../containers/ApplicationInfo";

import styles from "./styles.module.css";

function UserApplicationPage() {
  return (
    <div className={styles["referral-list-container"]}>
      <header>
        <Header />
      </header>
      <main>
        <article>
          <ApplicationInfo />
        </article>
      </main>
    </div>
  );
}

export default UserApplicationPage;
