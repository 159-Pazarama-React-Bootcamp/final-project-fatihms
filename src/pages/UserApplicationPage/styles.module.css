.uap-notfound {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  height: 100vh;
  width: 100%;
  background-color: #f5f5f5;
  font-size: 96px;
  font-weight: bold;
  color: darkred;
}
