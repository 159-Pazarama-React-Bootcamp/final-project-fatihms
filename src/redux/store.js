import { configureStore } from "@reduxjs/toolkit";

import codeReducer from "./code/codeSlice";

const store = configureStore({
  reducer: { code: codeReducer },
});

export default store;
