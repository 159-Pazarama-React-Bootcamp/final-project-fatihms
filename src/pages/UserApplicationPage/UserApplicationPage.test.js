/*eslint-disable */

import React from "react";

import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import { MemoryRouter as Router } from "react-router-dom";
import userEvent from "@testing-library/user-event";
import "@testing-library/jest-dom/extend-expect";

import { Provider } from "react-redux";
import store from "../../redux/store";

import UserApplicationPage from "./index";

describe("UserApplicationPage", () => {
  beforeEach(() => {
    render(
      <Provider store={store}>
        <Router>
          <UserApplicationPage />
        </Router>
      </Provider>
    );
  });
  test("UserApplicationPage renders correctly", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Router>
            <UserApplicationPage />
          </Router>
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
