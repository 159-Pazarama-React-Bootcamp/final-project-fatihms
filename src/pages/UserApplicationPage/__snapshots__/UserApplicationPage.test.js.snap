// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserApplicationPage UserApplicationPage renders correctly 1`] = `
<div
  className="loading"
>
  <div
    className="lds-ring"
  >
    <div>
       
    </div>
    <div>
       
    </div>
    <div>
       
    </div>
    <div>
       
    </div>
  </div>
</div>
`;
